### PR TITLE
Add examples for man and python script execution

### PR DESCRIPTION
**Pull request by srijanshetty**

### CODE REVIEW

**Comment by pawelduda** [comment]
Very nice. I will check it when I'm home and if everything is ok, I will merge it :)

File: scripts/fzf_python.md [hunk 1]
```diff
@@ -0,0 +1 @@
+echo '' | fzf --print-query --preview 'python -c {q}'
```
Comment (pawelduda) [inline scripts/fzf_python.md]: Could you please rename this file to `fzf_python.sh`?

**Comment by pawelduda** [comment]
@srijanshetty just one minor change and I'll merge it! Thank you!

**Comment by pawelduda** [comment]
@srijanshetty When it comes to the man example - I was also thinking about writing one that would navigate to flags as they're being typed. #1 contains a good starting point for that. Your simple example is good enough for now though. But I expect it to be replaced at some point - just FYI.

File: scripts/fzf_python.md (hunk 1 above)
Comment (srijanshetty) [reply scripts/fzf_python.md]: Done

**Comment by srijanshetty** [comment]
> @srijanshetty When it comes to the man example - I was also thinking about writing one that would navigate to flags as they're being typed. #1 contains a good starting point for that. Your simple example is good enough for now though. But I expect it to be replaced at some point - just FYI.

Definitely makes sense to do that, I thought that I would paste in a quick example.

While we're at it, how did you make those fancy gifs?

**Comment by pawelduda** [comment]
@srijanshetty 
> Definitely makes sense to do that, I thought that I would paste in a quick example.

Yes, quick example is better than no example ;)

> While we're at it, how did you make those fancy gifs?

I used [asciinema](https://asciinema.org/) and [asciicast2gif](https://github.com/asciinema/asciicast2gif). While it isn't the fastest solution (asciicast2gif takes a long time to generate even from shorter gifs), it was a quick solution to solve the problem I wanted before going to bed.

Also, thank you for the fixes!

**Pull request merged**